gallery/src/styles/Nivo/data.tsx

```diff
@@ -0,0 +1,335 @@
+/*
```

Comment:
This file should be included in the displayed code of each example.  I'd also split it up per-example.